docs/README.md

```diff
@@ -50,6 +50,42 @@ Prerequisite: In the API (packages/api) exists a corresponding GraphQL endpoint.
 4) Now the file `index.ts` will be generated automatically `in editor/src/client/api`
 5) Now you can import your desired endpoint in your .tsx file. See for example `subscriptionEditPanel.tsx`
 
+
+### Analytics
+In order to use analytics and send data to WePublish's Matomo, we prepared an analytics.js plugin that's available here:
+
+https://github.com/wepublish/analytics
+
+it integrates with https://github.com/DavidWells/analytics
+
+Usage:
+
+Import and initialise the tracker anywhere in the project
+
+    import  Analytics  from  'analytics'
+    import { matomoPlugin } from '@wepublish/analytics'
```

Comment:
Also here, I would not mention it's `matomoPlugin`... 
Ah that's what we actually create in analytics. Not sure if that's good idea. If we change the underlying solution `matomoPlugin` doesn't correspond well.

What about using plugin `WepublishPlugin`?

Reply:
Yes, let's call the plugin `wepublish` so when you initiate "analytics" with "wepublish" plugin.